avocado/utils/lv_utils.py

```diff
@@ -0,0 +1,517 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Author: Harish <harisrir@linux.vnet.ibm.com>
+# Copyright: 2016 IBM
+#
+# Based on the code by
+# Author: Plamen Dimitrov
+# Copyright: Intra2net AG 2012
```

Comment:
It's not mandatory, but it's nice to include a link to the project this file is based on, would you please add: https://github.com/autotest/autotest/blob/master/client/lv_utils.py